packages/account/src/Components/poinc/file-uploader-container/file-uploader-container.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Icon } from '@deriv/components';
+import { isDesktop, WS } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import FileUploader from '../../file-uploader';
+
+const PoincFileUploaderContainer = ({ document_type, is_description_enabled = true, getSocket, onFileDrop, onRef }) => {
+    const ref = React.useRef();
+    const getSocketFunc = getSocket ?? WS.getSocket;
+
+    React.useEffect(() => {
+        if (ref && typeof onRef === 'function') onRef(ref);
+        return () => onRef(undefined);
+    }, [onRef, ref]);
+
+    return (
+        <div className='account__file-uploader-section'>
+            {is_description_enabled && (
+                <ul className='account__file-uploader-list'>
+                    <li className='account__file-uploader-box'>
```

Comment:
@yauheni-kryzhyk-deriv can we map through these <li>s 🙂? they look identical

Reply:
you mean like this:
const description_items = ['Less than 8MB', 'JPEG JPG PNG PDF', 'Issued under your name with your current address', '1 - 6 months old']

description_items.map( text => 
                                <li className='account__file-uploader-box'>
                                      <Icon icon='IcClock' className='account__file-uploader-icon' size={24} />
                                      <div className='account__file-uploader-item'>
                                            <Localize i18n_default_text={text} />
                                   </div>
                               </li>)

?